src/gui/tray/UserModel.cpp

```diff
@@ -486,6 +486,11 @@ bool User::serverHasTalk() const
     return _account->hasTalk();
 }
 
+AccountApp *User::talkApp() const
+{
+    return _account->findApp(QLatin1String("spreed"));
```

Comment:
You're merely moving code around here, but good opportunity to use `QStringLiteral` instead of `QLatin1String` here.